packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -53,7 +53,10 @@ export class Tabs extends GOVUKFrontendComponent {
     /** @satisfies {NodeListOf<HTMLAnchorElement>} */
     const $tabs = $module.querySelectorAll('a.govuk-tabs__tab')
     if (!$tabs.length) {
-      return this
+      throw new ElementError(null, {
+        componentName: 'Tabs',
+        identifier: 'a.govuk-tabs__tab'
```

Comment:
Wondering if we need to show this _scoped_ via `$module` as it's not from `document` here?

```suggestion
        identifier: `[data-module="${Tabs.moduleName}"] a.govuk-tabs__tab`
```

Reply:
Went with the first one since we've been doing it that way so far, but we can always revisit when we review these.

Reply:
I asked @romaricpascal the same question on this thread here: https://github.com/alphagov/govuk-frontend/pull/4261#discussion_r1337169400

Will leave you two to decide what's best

Reply:
I think going for the `scope` thing will require quite a bit of investigation so I'd prefer we only look at it if needs arise.

Regarding the addition of `[data-module="${Tabs.moduleName}"]`, which new information does it bring in addition to:
- the name of the component that starts the error messages
- our classnames which usually refer to the component as well

Reply:
It came up [whilst discussing Checkboxes](https://github.com/alphagov/govuk-frontend/pull/4236#discussion_r1331778120) since realising it's only **Error summary**, **Radios**, **Checkboxes** and **Exit this page** that use `document.querySelector()` versus `$module.querySelector()` etc

So wondered if we should explain that or not, or undo previous scoping as part of https://github.com/alphagov/govuk-frontend/issues/4072

E.g. In previous PRs we've [scoped via `$module.hash target #${linkedElementId}`](https://github.com/alphagov/govuk-frontend/commit/fd7a8ed7e10d0dc8f4a362f0e7482839d0a43534#diff-fb3a25d42cbb3621d375cc34dc960447e84fa2a8729fd16e97e4c34edd9b17aeR66)

Reply:
Let's see once we have the whole list of errors for #4072. That'll give us a clear view of which component parts are referenced. And we can decide on which extra prefix we may need to better locate the faulty element on top of the name of the component that couldn't initialise.

Reply:
So drop the prefix in this PR like you've done in https://github.com/alphagov/govuk-frontend/pull/4261?